### ISSUE TITLE
Dog Kennels

### ISSUE THREAD

**Issue by Lucane1999** [LondonsBlueLine/tracker#367]
Hi team,

The dog kennel is great and I am already aware the ongoing effort to make each station has it own kennel.

I would love to see the removal of the husky in replacement for the Malinois dog and skin as seen in this link.
https://forum.cfx.re/t/how-to-german-shepherd-malinois-k9-dog-1-0-1/1065040

It would then be amazing to lock each dog to the division of DSU.

DSU - German Shep
DCFD - Retriever
DSU A - Malinois

Therefore people are committed to unlocking extra dogs per each division.

**Comment by telboycafc** [comment]
One for jack I think, beyond my ability.

**Comment by jxckUK** [comment]
@WolfRanger88 - Could you take a look at this latest one and see if it can go in?

**Comment by WolfRanger88** [comment]
@jxckUK I believe I may be able to find a work around on this to be able to choose the dog eup and textures in similar fashion to how we work the lockers.

**Comment by Lucane1999** [comment]
Yes it would be great if the Husky was renamed to Malinois (german shep model with the eup as the Malinois).
And then to lock them similar to garage. so that they are role specific.

DSU - German Shep
DCFD - Retriever
DSU A - Malinois
